app/controllers/git_api/tracks_controller.rb

```diff
@@ -5,20 +5,25 @@ class TracksController < ApplicationController
     layout false
 
     def creation_issues
-      issues = fetch_issues('type/new-exercise')
+      issues = fetch_issues(%w(type/new-exercise status/help-wanted), %w(OPEN))
       render json: issues
     end
 
+    def creation_issues_count
+      issues = fetch_issues(%w(type/new-exercise), %w(OPEN CLOSED))
```

Comment:
I'm a little uncertain on the naming of these methods. I would expect `creation_issues` and `creation_issues_count` do the same things, but they don't. So maybe:
```
open_creation_issues
num_creation_issues
open_improve_issues
```

Reply:
I like that!